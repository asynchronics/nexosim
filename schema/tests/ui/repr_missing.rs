use schemars::Message_repr;

#[derive(Message_repr)]
pub enum Enum {
    Unit,
}

fn main() {}
