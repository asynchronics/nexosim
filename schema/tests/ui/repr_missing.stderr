error: Message_repr: missing #[repr(...)] attribute
 --> $DIR/repr_missing.rs:3:10
  |
3 | #[derive(Message_repr)]
  |          ^^^^^^^^^^^^^^^
  |
  = note: this error originates in the derive macro `Message_repr` (in Nightly builds, run with -Z macro-backtrace for more info)
